src/react-apps/applications/service-development/src/features/administration/components/Administration.tsx

```diff
@@ -0,0 +1,272 @@
+      `${altinnWindow.location.origin}/designerapi/Repository/GetRepository?owner=${org}&repository=${service}`);
+    handleServiceInformationActionDispatchers.fetchInitialCommit(
+      `${altinnWindow.location.origin}/designerapi/Repository/GetInitialCommit?owner=${org}&repository=${service}`);
+    handleServiceInformationActionDispatchers.fetchServiceName(
+      `${altinnWindow.location.origin}/designer/${org}/${service}/Text/GetServiceName`);
+    handleServiceInformationActionDispatchers.fetchServiceDescription(
+      `${altinnWindow.location.origin}/designer/${org}/${service}/Config/GetServiceDescription`);
+  }
+
+  public onServiceNameChanged = (event: any) => {
+    this.setState({ serviceName: event.target.value, serviceNameAnchorEl: null });
+  }
+
+  public handleEditServiceName = () => {
+    this.setState({ editServiceName: true });
+  }
+
+  public onBlurServiceName = () => {
+    if (this.state.editServiceName && (!this.state.serviceName || this.state.serviceName === '')) {
+      this.setState({
+        serviceNameAnchorEl: document.getElementById('service-name'),
+      });
+    }
+
+    if (this.state.editServiceName) {
+      const altinnWindow: any = window;
+      const { org, service } = altinnWindow;
+      // tslint:disable-next-line:max-line-length
+      handleServiceInformationActionDispatchers.saveServiceName(`${altinnWindow.location.origin}/designer/${org}/${service}/Text/SetServiceName`, this.state.serviceName);
+      this.setState({ editServiceName: false });
+    }
+  }
+
+  public onServiceDescriptionChanged = (event: any) => {
+    this.setState({ serviceDescription: event.target.value, editServiceDescription: true });
+  }
+
+  public onBlurServiceDescription = () => {
+    if (this.state.editServiceDescription) {
+      const altinnWindow: any = window;
+      const { org, service } = altinnWindow;
+      // tslint:disable-next-line:max-line-length
+      handleServiceInformationActionDispatchers.saveServiceDescription(`${altinnWindow.location.origin}/designer/${org}/${service}/Config/SetServiceDescription`, this.state.serviceDescription);
+      this.setState({ editServiceDescription: false });
+    }
+  }
+
+  public render() {
+    const { classes } = this.props;
+
+    return (
+      <div className={classes.mainLayout}>
+        <VersionControlHeader language={this.props.language} />
+        {this.props.service || this.props.serviceName === null || this.props.serviceDescription === null ?
+          <Grid container={true} className={classes.layout}>
+            <Grid item={true} className={classes.mainStyle} md={12}>
+              <Typography className={classes.headerStyle}>
+                {getLanguageFromKey('administration.administration', this.props.language)}
+              </Typography>
+            </Grid>
+            <Grid item={true} className={classes.mainStyle} md={8}>
+              <AltinnInputField
+                id={'service-name'}
```

Comment:
Curly brackets are unnecesarry when you define the 'property'.

Reply:
But they are so cool 😎

Reply:
fixed